src/GovUk.Education.ExploreEducationStatistics.Publisher.Model/ViewModels/ReleaseTypeViewModel.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System;
+﻿#nullable enable
+using System;
 
 namespace GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels
 {
-    public class ReleaseTypeViewModel
+    public record ReleaseTypeViewModel
```

Comment:
Just letting you know we discussed this yesterday as there's a need to introduce a new ReleaseType soon, and said that `ReleaseType` could be replaced with an enum of types, since there a very few and new ones are rarely introduced. So this could potentially be dropped in future to be replaced with a label for the type.

Reply:
Cool cool :+1: always thought it was a little bit unwieldy in its current form. Are we thinking of dropping the database table in favour of storing it as enum strings?

Reply:
Yes that's what we chatted about, as they change so rarely. This didn't have anything much to do with your PR but thought I'd mention it here when I saw it! 🙂